wp-multi-network/includes/classes/class-wp-ms-network-command.php

```diff
@@ -260,7 +260,7 @@ public function list_( $args, $assoc_args ) {
 	 * @param array $assoc_args Associative CLI arguments.
 	 */
 	public function plugin( $args, $assoc_args ) {
-		$this->fetcher = new \WP_CLI\Fetchers\Plugin();
+		$fetchers_plugin = new \WP_CLI\Fetchers\Plugin();
```

Comment:
I believe that this should be a var and not a class member (PHPStan also didn't like it :-))